pages/index.vue

```diff
@@ -332,19 +302,13 @@ export default {
   position: relative;
   z-index: 5;
   width: 100%;
-  // margin-top: 50vh;
+  background-color: #181829;
```

Comment:
Not familiar with the code base but can we add this colour to a variable?

Reply:
Yeah it should be, I did merge Will's work into my branch and maybe this one didn't auto merge correctly. I'll look into it now